src/components/CR8 - GalleryText/GalleryText.vue

```diff
@@ -0,0 +1,234 @@
+<template>
+    <div class="gallery-text">
+        <!-- Title -->
+        <h2 class="title">Our Ecosystem</h2>
+
+        <!-- Subtitle w/ Arrow -->
+        <div class="subtitle">
+            <span class="subtitle-text">We develop and produce original content for all genres and formats.</span>
+            <span class="prompt">
+                <span class="prompt-text" >EXPLORE</span>
+                <svg-arrow-right class="arrow" />
+            </span>
+        </div>
+        
+        <!-- Block Links -->
+        <div class="text">
+            <nuxt-link
+                v-for="(company, index) in companies"
+                :key="`${index}-${company.name}`" 
+                class="company-link"  
+                :to="`/${company.name}`" tag="span"
+            >
+                <component 
+                    :is="`logo-${company.name}`"
+                    class="company-logo"
+                    @mouseover="set(company)" 
+                    @mouseout="set(null)" />
+                {{company.text}}
+            </nuxt-link>
+        </div>
+
+        <!-- Images absolute -->
+        <transition
+            v-for="(image, index) in images"
+            :key="image.id"
+            name="fade"
+            mode="out-in"
+        >
+            <wp-image
+                v-show="index == activeImage"
+                class="image"
+                :image="image"
+                mode="fullbleed"
+            />
+        </transition>
+
+        <!-- Bottom Text -->
+        <p class="end-text">These diverse talent are connected by a creative singularity.</p>
+        <p class="end-text">Together we are <logo-makemake class="company-logo" /></p>
+
+    </div>
+</template>
+
+<script>
+import WpImage from "@/components/global/WpImage"
+import NuxtLink from "@/components/global/NuxtLink"
+import SvgArrowRight from "@/assets/svgs/arrow-right.svg"
+
+// Company Logos
+import LogoRockPaperScissors from '@/assets/svgs/companies/logo-rock-paper-scissors.svg'
+import LogoElastic from '@/assets/svgs/companies/logo-elastic.svg'
+import LogoA52 from '@/assets/svgs/companies/logo-a52.svg'
+import LogoPrimary from '@/assets/svgs/companies/logo-primary.svg'
+import LogoJax from '@/assets/svgs/companies/logo-jax.svg'
+import LogoIndestructible from '@/assets/svgs/companies/logo-indestructible.svg'
+
+import LogoMakemake from '@/assets/svgs/companies/logo-makemake.svg'
+
+export default {
+    components: { 
+        WpImage, NuxtLink, SvgArrowRight, 
+        //Company Logos
+        LogoRockPaperScissors, LogoElastic, LogoA52, LogoPrimary, LogoJax, LogoIndestructible, LogoMakemake
+    },
```

Comment:
These should all be on a new line each. Reason being that if I change them, Git will only show the line changed, and it's harder to know which part of the line.